Formula/ib.rb

```diff
@@ -0,0 +1,16 @@
+class Ib < Formula
+  desc "C/C++ build tool that automatically resolves included source"
+  homepage "https://github.com/JasonL9000/ib"
+  url "https://github.com/JasonL9000/ib/archive/0.7.tar.gz"
+  sha256 "2bb8c6b456fb39d0b63fc40d96e4f494c3599ebc39ae1609601b9be1b2ef036f"
+  depends_on :python
+
+  def install
+    prefix.install "common.cfg", "debug.cfg", "release.cfg", "asan.cfg", "__ib__"
+    bin.install "ib"
+  end
+
+  test do
+    system "#{bin}/ib", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. In this case you can `.write` out a `hello.cc` like in the example. Thanks!